### DISCUSSION TITLE
Mutants and Masterminds 3rd edition

### DISCUSSION THREAD

**Discussion by Asmardos** [BahamutDragon/pcgen#1200] [Q&A] [answered]
Is there any idea when Mutants and Masterminds 3rd edition will be available?
We play our first round in november and i can't afford hero lab because i'm cronically broke.

**Comment by BahamutDragon** [comment, marked as answer]
Being chronically broke is not a desirable position to be in. Not a life coach myself mate, but perhaps taking control of your finances by some sort of program might be beneficial. Lacking essentials is never fun. At the very least managing debt and using the snowball method to pay off debts should be a goal, once the debts are managed, saving up for six months of expenses would be your next goal. Life loves to throw surprise curve balls.

However, I doubt you came here seeking financial advice. Taking a look at the book and notes I have... The book is untested but barring any critical bugs should be playable with whatever bugs are to be found. The outstanding note is a lack of a character sheet. Currently it is using a default fantasy sheet. Will you be using PCGen 6.08 or the development 6.09 series?

```
Mutants and Masterminds 3rd edition - Needs OS
 Green Ronin - Core Rulebook (75% complete)

```

Do you have a character sheet in mind and which format you will be using?
The output choices are HTML, PDF, or simple text.

Text is usually the easiest to churn out. It's simplistic. PDF usually is the most work, but has the best visual ascetics in my opinion.

I can likely do up a text sheet pretty quickly by this Sunday.

If you are willing to test the set and sheet that would be fantastic!
